### PR TITLE
Project Add API: separate lane and platform

### PR DESCRIPTION
**Pull request by anshulagl**
- [x] I have run `rspec` and corrected all errors
- [x] I have run `rubocop` and corrected all errors
- [x] I have run `npm run test` and corrected all errors
- [ ] I have tested this change locally and tried to launch the server as well as access a project, and with that at least one build
- [x] If there is an existing issue, make sure to add `Fixes ...` as part of the PR body to reference the issue you're solving

Fixes Issue #1052 , needs updating the frontend to send platform as a separate parameter in the add project API request.

### CODE REVIEW

**Comment by anshulagl** [comment]
I signed it!

File: app/features-json/project_json_controller.rb [hunk 1]
```diff
@@ -33,8 +33,9 @@ class ProjectJSONController < APIController
       end
 
       repo_config = GitHubRepoConfig.from_octokit_repo!(repo: selected_repo)
-
-      platform, lane = params["lane"].split(" ") # Split "ios test_lane"
+      lane = params["lane"]
+      # If platform is empty string, means no platform
+      platform = params["platform"] == "" ? :no_platform : params["platform"]
```
Comment (nakhbari) [inline app/features-json/project_json_controller.rb]: won't this be the same error since the backend will be getting `no_platform` as a platform name?

Comment (anshulagl) [reply app/features-json/project_json_controller.rb]: You are right. The issue is not while configuring the project, but during the build process. https://github.com/fastlane/ci/pull/951 automatically fixed the issue for manual builds.

**Comment by anshulagl** [comment]
Closing this PR due to corrupted commits (unknown author CLA problems). The issue also turned out to be in build system and not in project configuration.

**Pull request closed without merging**